### PR TITLE
zh-translation: docs/root/configuration/http/http_filters/aws_lambda_…

### PR DESCRIPTION
**Pull request by vgbhfive**
…filter.rst

For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes: docs/root/configuration/http/http_filters/aws_lambda_filter.rst
Release Notes:
[Optional Runtime guard:]
[Optional Fixes #Issue] #363
[Optional Deprecated:]

### CODE REVIEW

File: docs/root/configuration/http/http_filters/aws_lambda_filter.rst [hunk 1]
```diff
@@ -4,23 +4,18 @@
...
+  AWS Lambda 过滤器目前正在积极开发中。
 
-The HTTP AWS Lambda filter is used to trigger an AWS Lambda function from a standard HTTP/1.x or HTTP/2 request.
-It supports a few options to control whether to pass through the HTTP request payload as is or to wrap it in a JSON
-schema.
+HTTP AWS Lambda 过滤器用于从标准 HTTP/1.x 或 HTTP/2 请求中触发 AWS Lambda 函数。
 
-If :ref:`payload_passthrough <envoy_v3_api_field_extensions.filters.http.aws_lambda.v3.Config.payload_passthrough>` is set to
-``true``, then the payload is sent to Lambda without any transformations.
-*Note*: This means you lose access to all the HTTP headers in the Lambda function.
```
Comment (addozhang) [inline docs/root/configuration/http/http_filters/aws_lambda_filter.rst:20]: 这行没翻译？

File: docs/root/configuration/http/http_filters/aws_lambda_filter.rst [hunk 2]
```diff
@@ -33,22 +28,21 @@ is set to ``false``, then the HTTP request is transformed to a JSON payload with
...
     }
 
-- ``rawPath`` is the HTTP request resource path (including the query string)
-- ``method`` is the HTTP request method. For example ``GET``, ``PUT``, etc.
-- ``headers`` are the HTTP request headers. If multiple headers share the same name, their values are
-  coalesced into a single comma-separated value.
-- ``queryStringParameters`` are the HTTP request query string parameters. If multiple parameters share the same name,
-  the last one wins. That is, parameters are _not_ coalesced into a single value if they share the same key name.
-- ``body`` the body of the HTTP request is base64-encoded by the filter if the ``content-type`` header exists and is _not_ one of the following:
+- ``rawPath`` 是 HTTP 请求支援路径（包含查询字符串）
```
Comment (addozhang) [inline docs/root/configuration/http/http_filters/aws_lambda_filter.rst]:
```suggestion
- ``rawPath`` 是 HTTP 请求资源路径（包含查询字符串）
```

File: docs/root/configuration/http/http_filters/aws_lambda_filter.rst [hunk 3]
```diff
@@ -60,30 +54,30 @@ On the other end, the response of the Lambda function must conform to the follow
...
 
-- The ``statusCode`` field is an integer used as the HTTP response code. If this key is missing, Envoy returns a ``200
-  OK``.
-- The ``headers`` are used as the HTTP response headers.
-- The ``cookies`` are used as ``Set-Cookie`` response headers. Unlike the request headers, cookies are _not_ part of the
-  response headers because the ``Set-Cookie`` header cannot contain more than one value per the `RFC`_. Therefore, Each
-  key/value pair in this JSON array will translate to a single ``Set-Cookie`` header.
-- The ``body`` is base64-decoded if it is marked as base64-encoded and sent as the body of the HTTP response.
+- ``statusCode`` 字段是用作 HTTP 响应码的整数。如果这个键消失，Envoy 会返回一个 ``200 OK``。
+-  ``headers`` 用作 HTTP 响应头。
```
Comment (addozhang) [inline docs/root/configuration/http/http_filters/aws_lambda_filter.rst]:
```suggestion
- ``headers`` 用作 HTTP 响应头。
```

File: docs/root/configuration/http/http_filters/aws_lambda_filter.rst [hunk 4]
```diff
@@ -183,17 +182,19 @@ An example with the Lambda metadata applied to a weighted-cluster:
...
+:ref:`stat 前置 <envoy_api_field_config.filter.network.http_connection_manager.v2.HttpConnectionManager.stat_prefix>` 来自于拥有的 HTTP 连接管理器。
 
 .. csv-table::
-  :header: Name, Type, Description
+  :header: 名称, 类型, 描述
   :widths: 1, 1, 2
 
-  server_error, Counter, Total requests that returned invalid JSON response (see :ref:`payload_passthrough <envoy_api_msg_config.filter.http.aws_lambda.v2alpha.config>`)
-  upstream_rq_payload_size, Histogram, Size in bytes of the request after JSON-tranformation (if any).
+  server_error, Counter, 返回无效 JSON 响应的请求总数（看 :ref:`payload_passthrough <envoy_api_msg_config.filter.http.aws_lambda.v2alpha.config>` ）。
```
Comment (addozhang) [inline docs/root/configuration/http/http_filters/aws_lambda_filter.rst]:
```suggestion
  server_error, Counter, 返回无效 JSON 响应的请求总数（见 :ref:`payload_passthrough <envoy_api_msg_config.filter.http.aws_lambda.v2alpha.config>` ）。
```

**Review by addozhang — COMMENTED** [review]
不错的翻译，添加了点建议，辛苦再看下。

File: docs/root/configuration/http/http_filters/aws_lambda_filter.rst (hunk 1 above)
Comment (vgbhfive) [reply docs/root/configuration/http/http_filters/aws_lambda_filter.rst:20]: 是我忘记删除原文了😂

**Review by addozhang — APPROVED** [review]
/LGTM

File: docs/root/configuration/http/http_filters/aws_lambda_filter.rst (hunk 1 above)
Comment (helight) [reply docs/root/configuration/http/http_filters/aws_lambda_filter.rst:20]: *Note*: This means you lose access to all the HTTP headers in the Lambda function.
这行没有翻译呀

File: docs/root/configuration/http/http_filters/aws_lambda_filter.rst [hunk 5]
```diff
@@ -4,23 +4,18 @@
...
 
-The HTTP AWS Lambda filter is used to trigger an AWS Lambda function from a standard HTTP/1.x or HTTP/2 request.
-It supports a few options to control whether to pass through the HTTP request payload as is or to wrap it in a JSON
-schema.
+HTTP AWS Lambda 过滤器用于从标准 HTTP/1.x 或 HTTP/2 请求中触发 AWS Lambda 函数。
 
-If :ref:`payload_passthrough <envoy_v3_api_field_extensions.filters.http.aws_lambda.v3.Config.payload_passthrough>` is set to
-``true``, then the payload is sent to Lambda without any transformations.
-*Note*: This means you lose access to all the HTTP headers in the Lambda function.
+如果 :ref:`payload_passthrough <envoy_v3_api_field_extensions.filters.http.aws_lambda.v3.Config.payload_passthrough>` 被设置为 ``true``，然后将负载发送到 Lambda，而不进行任何转换。
```
Comment (helight) [inline docs/root/configuration/http/http_filters/aws_lambda_filter.rst]:
```suggestion
如果 :ref:`payload_passthrough <envoy_v3_api_field_extensions.filters.http.aws_lambda.v3.Config.payload_passthrough>` 被设置为 ``true``，那么负载将会被发送到 Lambda 函数，而不进行任何转换。
```

**Review by helight — COMMENTED** [review]
一点意见

File: docs/root/configuration/http/http_filters/aws_lambda_filter.rst (hunk 5 above)
Comment (vgbhfive) [reply docs/root/configuration/http/http_filters/aws_lambda_filter.rst]: 是我看错了😂，抱歉

**Review by helight — APPROVED** [review]
LGTM

**Pull request merged**